django/core/api/views.py

```diff
@@ -13,3 +17,8 @@ def get(self, request):
         """
         data = {"commit": settings.VERSION_SHA, "version": settings.VERSION_NAME}
         return Response(data)
+
+
+def custom_error(request, *args, **argv):
```

Comment:
maybe this need a comment on why we're doing this

Reply:
Apparently, custom error handling example uses function and not class based design :
https://www.django-rest-framework.org/api-guide/exceptions/#custom-exception-handling